src/Microsoft.Azure.SignalR.Management/Microsoft.Azure.SignalR.Management.csproj

```diff
@@ -38,7 +38,6 @@
   </Target>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0'">
-    <PackageReference Include="Microsoft.AspNetCore.SignalR" Version="$(MicrosoftAspNetCoreSignalRPackageVersion)" />
```

Comment:
for netstandard2.0 I think this dep is required

Reply:
Moved to `common` as well. See https://github.com/Azure/azure-signalr/blob/dev/src/Microsoft.Azure.SignalR.Common/Microsoft.Azure.SignalR.Common.csproj#L20-L22